packages/cxl-ui/src/components/cxl-jw-player/mixins/NextUpMixin.js

```diff
@@ -0,0 +1,42 @@
+import { html, render } from 'lit';
+import style from '../../../styles/cxl-jw-player/cxl-jw-player-next-up-css';
+export function NextUpMixin(BaseClass) {
+  class Mixin extends BaseClass {
+    __nextUpCTA;
```

Comment:
If we look at Lit, https://lit.dev/docs/api/decorators/ shows all "internal" stuff is single-underscore. I think we should follow upstream for such.

Ideally there could be an ESLint rule to auto-fix.

Reply:
I believe double underscore represents private methods and properties while single underscore represents private methods and properties.

Lit does use private properties internally (https://github.com/lit/lit/blob/b0c3f82ef0f97326a205e77e7e1043b75a5cc53f/packages/lit-html/src/lit-html.ts#L651).

Unless I'm certain I'm certain a subclass should be able to override a method or property, I tend to opt for private.

I don't mind switching to protected, and I can start with this mixin. Refactoring the rest of the component will take some time.

We were trying to ship this component soon for limited use on the dashboard for A/B testing.

Reply:
I've refactored all methods and properties to protected.